src/Result.ts

```diff
@@ -1,13 +1,14 @@
 import { analyze } from './AxePage';
 import { showStory } from './StorybookPage';
-import type { Result as AxeResult, NodeResult } from 'axe-core';
+import type { Result as AxeResult } from 'axe-core';
 import type { Page } from 'playwright';
 import { getDisabledRules, ProcessedStory } from './ProcessedStory';
 
 /**
  * Violations reported by Axe for a story.
  */
 export type Result = {
+  component: string,
```

Comment:
We now need the component name in the result, to help make formatting the result easier.